<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowDatabase</class>
 <widget class="QWidget" name="WindowDatabase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>529</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableView" name="GamesTable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>371</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="editGame">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>40</y>
     <width>120</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveAndClose">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>260</y>
     <width>120</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Save &amp;&amp; Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteGame">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>160</y>
     <width>120</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="filterGames">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>120</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Filter Games</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exportGames">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>210</y>
     <width>120</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Export Games</string>
   </property>
  </widget>
  <widget class="QPushButton" name="importGames">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>210</y>
     <width>120</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Import Games</string>
   </property>
  </widget>
  <widget class="QPushButton" name="openGame">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>10</y>
     <width>120</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </widget>
  <widget class="QPushButton" name="copyGame">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>90</y>
     <width>120</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pasteGame">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>120</y>
     <width>120</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveChanges">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>120</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Changes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeOnly">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>260</y>
     <width>120</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="undoChanges">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>260</y>
     <width>120</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Undo Changes</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
